Code/GraphMol/MolPickler.cpp

```diff
@@ -94,6 +94,15 @@ std::mutex &GetPropMutex() {
   return propmutex_get();
 }
 #endif
+
+void write_sstream_to_stream(std::ostream &outStream,
+                             const std::stringstream &toWrite) {
+  auto ts = toWrite.str();
```

Comment:
Is there a better way to get the size of a string stream?  Perhaps seek(0) then tellg?

Reply:
we need to get the string anyway in order to write it out

Reply:
I think you can do

```
outStream << toWrite;
```

But if it's a premature optimization, that's ok.